src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -184,4 +188,16 @@ public void shutdown() {
             ioTCoreClient.stop();
         }
     }
+
+    private boolean skipUpdatingMqttTopicMapping(WhatHappened whatHappened, Node node) {
+        if (whatHappened == WhatHappened.timestampUpdated || whatHappened == WhatHappened.interiorAdded) {
+            return true;
+        }
+        if (node != null && BROKER_URI_KEY.equals(node.getName())) {
```

Comment:
I think there's a subtle regression where we aren't setting the mqtt config on WhatHappened.initialized (node == null)

Reply:
please disregard the above, i misread this 😅